modules/local/linux/cut.nf

```diff
@@ -1,35 +1,28 @@
-include { initOptions; saveFiles; getSoftwareName; getProcessName } from '../common/functions'
-
-params.options   = [:]
-options          = initOptions(params.options)
-options.command  = params.options.command ?: ''
-options.ext      = params.options.ext ?: ''
-
 process CUT {
     tag "$meta.id"
-    label 'process_low'
-    publishDir "${params.outdir}",
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+    label 'process_min'
 
-    conda (params.enable_conda ? null : null)
-    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img"
-    } else {
-        container "biocontainers/biocontainers:v1.2.0_cv1"
-    }
+    conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
 
     input:
     tuple val(meta), path(input)
 
     output:
     tuple val(meta), path("*.cut.*"), emit: file
 
```

Comment:
missing versions.yml

Reply:
@chris-cheshire this is the only one left as far as i see. any updates here?

Reply:
Okay @apeltzer and @FriederikeHanssen, I have added versioning for both cut and sort now, thanks for your solution!